apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/PDFGenerator.kt

```diff
@@ -139,7 +140,28 @@ class PDFGenerator(
 
         return brevbakerService
             .genererPdf(brev.id, brevRequest)
-            .also {
+            .let {
+                // TODO: finne en bedre plass for dette?
+                when (brev.brevkoder) {
+                    Brevkoder.OMS_INNTEKTSJUSTERING_VARSEL -> {
+                        val vedtaksbrev =
+                            brevbakerService.genererPdf(
+                                brev.id,
+                                BrevbakerRequest.fra(
+                                    EtterlatteBrevKode.OMSTILLINGSSTOENAD_INNTEKTSJUSTERING_VEDTAK,
```

Comment:
Vi må finne ut om det skal være en egen mal for dette. Jeg tror ikke det. Så det kan være det blir litt vanskjelig å utlede om det skal være varsel basert på brevkode alene. Mulig det må til et flagg i brevRequest eller noe sånt 🤔